src/api/søknadApi.ts

```diff
@@ -19,6 +19,10 @@ interface Avvist {
 
 export type LukkSøknadBodyTypes = Trukket | Bortfalt | Avvist;
 
+export type AvslagManglendeDokType = {
+    fritekst: string;
+};
```

Comment:
Vi foretrekker `interface` for slike.

```suggestion
export interface AvslagManglendeDokBody {
    fritekst: string;
};
```